pkg/injector/sidecar/consts.go

```diff
@@ -15,21 +15,23 @@ limitations under the License.
 package sidecar
 
 const (
-	SidecarContainerName          = "daprd" // Name of the Dapr sidecar container
-	SidecarHTTPPort               = 3500
-	SidecarAPIGRPCPort            = 50001
-	SidecarInternalGRPCPort       = 50002
-	SidecarPublicPort             = 3501
-	SidecarHTTPPortName           = "dapr-http"
-	SidecarGRPCPortName           = "dapr-grpc"
-	SidecarInternalGRPCPortName   = "dapr-internal"
-	SidecarMetricsPortName        = "dapr-metrics"
-	SidecarDebugPortName          = "dapr-debug"
-	SidecarHealthzPath            = "healthz"
-	APIVersionV1                  = "v1.0"
-	UnixDomainSocketVolume        = "dapr-unix-domain-socket" // Name of the Unix domain socket volume.
-	UserContainerDaprHTTPPortName = "DAPR_HTTP_PORT"          // Name of the variable exposed to the app containing the Dapr HTTP port.
-	UserContainerDaprGRPCPortName = "DAPR_GRPC_PORT"          // Name of the variable exposed to the app containing the Dapr gRPC port.
-	ContainersPath                = "/spec/containers"
-	KubernetesMountPath           = "/var/run/secrets/kubernetes.io/serviceaccount" // Mount path for the Kubernetes service account volume.
```

Comment:
Is this a breaking change?

Reply:
No, unless people are doing some really weird stuff.

Here's how it works:

- A new bound token is mounted to `/var/run/secrets/dapr.io/sentry` regardless. That has no permissions to the K8s APIs and it's only used for the runtime to authenticate itself with Sentry. Applications should not have attempted to communicate with Sentry regardless.
- *If and only if* `automountServiceAccountToken: true` (the default value), then a service account token with permissions to talk to the K8s APIs is mounted at the same path as before `/var/run/secrets/kubernetes.io/serviceaccount`. This token is also used by Dapr components that talk to K8s (such as the K8s secret store - but only for accessing secrets at runtime, and not for component secrets - or K8s binding). The Dapr runtime doesn't care about this token at all, whether it's there or not. Some Dapr components may require it. It's up to the developers to decide to add `automountServiceAccountToken: false` to the deployment if they don't need it (by default Dapr does not alter that value in either direction).

Reply:
So the constant has been removed because the runtime doesn't need it anymore. But components still know where to find it (https://github.com/dapr/components-contrib/blob/master/internal/authentication/kubernetes/client.go)